src/ssl.c

```diff
@@ -18299,7 +18299,8 @@ size_t wolfSSL_get_client_random(const WOLFSSL* ssl, unsigned char* out,
     #endif
 #endif
 #ifndef NO_SHA
-        if (XSTRCMP(evp_md, "SHA") == 0) {
+        if (XSTRCMP(evp_md, "SHA") == 0 ||
```

Comment:
Fairly certain Hayden already put up a fix for these here: https://github.com/wolfSSL/wolfssl/pull/5567/files

Reply:
Woops, he sure did.  I didn't see that PR.  I'll check with Hayden, may make more sense to merge the duplicate fix into that PR.